snorkel/labeling/model/label_model.py

```diff
@@ -354,7 +355,7 @@ def predict_proba(self, L: sparse.spmatrix) -> np.ndarray:
         >>> label_model = LabelModel()
         >>> label_model.train_model(L)
         >>> label_model.predict_proba(L)
-        np.array([1.0, 0.0], [0.0, 1.0], [1.0, 0.0])
+        np.array([[1.0, 0.0], [0.0, 1.0], [1.0, 0.0]])
```

Comment:
just a note: you can test these with doctest (https://docs.python.org/3/library/doctest.html). not set up to run automatically with pytest yet, but it's on the list.